wrapper/hosts.py

```diff
@@ -483,6 +483,12 @@ def prepare_command(self, data, v2v_args, v2v_env, v2v_caps):
             v2v_args.extend([
                 '-oo', 'verify-server-certificate=false'
                 ])
+        if 'luks_keys_files' in data:
+            for luks_key in data['luks_keys_files']:
+                v2v_args.extend([
+                    '--key',
+                    '%s:file:%s' % (luks_key['device'], luks_key['filename'])
+                ])
```

Comment:
These changes should go to `prepare_command()` in `virt_v2v_wrapper.py` instead of into every host class.

Reply:
Done